### PR TITLE
Added Card Components [P0]

### PR DESCRIPTION
**Pull request by samflab**
please review it. thanks.
You can checkout the screenshots here:
Deployed at: https://deploy-preview-7--azure-ui.netlify.app/card/card
image

### CODE REVIEW

File: card/card.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  z-index: 0;
+}
+
+.price {
+  color: var(--card-price);
+  font-weight: 600;
+  font-size: 1.2rem;
+}
+
+del {
```
Comment (xsaurabhsuthar) [inline card/card.css]: use class for selection instead of tag.

**Review by xsaurabhsuthar — COMMENTED** [review]
image
This card has so much empty space. You can decrease width for this.

File: card/card.css (hunk 1 above)
Comment (samflab) [reply card/card.css]: done. thank you

Comment (xsaurabhsuthar) [reply card/card.css]: Looks perfect.

**Review by xsaurabhsuthar — APPROVED** [review]

**Review by Nishant-96 — APPROVED** [review]

**Review by NtshVrm — APPROVED** [review]

**Review by NtshVrm — APPROVED** [review]

**Pull request merged**